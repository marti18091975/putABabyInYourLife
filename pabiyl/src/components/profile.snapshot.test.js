import renderer from 'react-test-renderer';
import React from 'react';
import Profile from './Profile';
import { BrowserRouter as Router } from 'react-router-dom';

describe('App snapshot', () => {

    const tree = renderer.create(
        <Router>
            <Profile />
        </Router>
    );

    it('should match', () => {
        expect(tree.toJSON()).toMatchSnapshot();
    });
});