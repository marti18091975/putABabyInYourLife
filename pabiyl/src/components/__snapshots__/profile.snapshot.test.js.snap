// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should match 1`] = `
<main
  className="main__detail"
>
  <div
    className="main__detail--top"
  >
    <div
      className="left-side"
    >
      <section
        className="container__info"
      >
        <div
          className="text__detail name__detail"
        >
           
        </div>
        <div
          className="container__bottom--detail"
        >
          <div
            className="container"
          >
            <div
              className="container__detail"
            >
              <img
                alt="gender"
                className="icon__detail"
                src="https://image.flaticon.com/icons/svg/3260/3260246.svg"
              />
              <div
                className="text__detail"
              />
            </div>
            <div
              className="container__detail"
            >
              <img
                alt="age"
                className="icon__detail"
                src="https://image.flaticon.com/icons/svg/3021/3021790.svg"
              />
              <div
                className="text__detail"
              />
            </div>
          </div>
          <div
            className="container"
          >
            <div
              className="container__detail"
            >
              <img
                alt="City"
                className="icon__detail"
                src="https://image.flaticon.com/icons/svg/888/888063.svg"
              />
              <div
                className="text__detail"
              />
            </div>
            <div
              className="container__detail"
            >
              <img
                alt="country"
                className="icon__detail"
                src="https://image.flaticon.com/icons/svg/3455/3455212.svg"
              />
              <div
                className="text__detail"
              />
            </div>
          </div>
          <div
            className="container group-three"
          >
            <div
              className="container__detail"
            >
              <img
                alt="civil state"
                className="icon__detail"
                src="https://image.flaticon.com/icons/svg/1102/1102457.svg"
              />
              <div
                className="text__detail"
              />
            </div>
            <div
              className="container__detail"
            >
              <img
                alt="sons"
                className="icon__detail"
                src="https://image.flaticon.com/icons/svg/1761/1761430.svg"
              />
              <div
                className="text__detail"
              />
            </div>
            <div
              className="container__detail"
            >
              <img
                alt="job"
                className="icon__detail"
                src="https://image.flaticon.com/icons/svg/1063/1063196.svg"
              />
              <div
                className="text__detail"
              />
            </div>
          </div>
        </div>
        <div
          className="buttons bt__inside"
        >
          <a
            aria-current={null}
            className="navLink__edit"
            href="/profileEditor"
            onClick={[Function]}
          >
            <button
              className="btn__navLink"
            >
              EDIT
            </button>
          </a>
        </div>
      </section>
      <section
        className="left-side__bottom"
      >
        <img
          alt=""
          className="image__secundary"
        />
        <img
          alt=""
          className="image__secundary"
        />
      </section>
    </div>
    <section
      className="right-side__detail"
    >
      <img
        alt=""
        className="image__main"
      />
      <div
        className="buttons bt__outside"
      >
        <a
          aria-current={null}
          className="navLink__edit"
          href="/profileEditor"
          onClick={[Function]}
        >
          <button
            className="btn__navLink--outside"
          >
            EDIT
          </button>
        </a>
      </div>
    </section>
  </div>
  <div
    className="presentation"
  />
</main>
`;
